src/extension/features/accounts/hide-memos/components/hide-memos-button.jsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import { l10n, getToolkitStorageKey } from 'toolkit/extension/utils/toolkit';
+
+export const HideMemosButton = ({ toggleHiddenState }) => {
+  const notHidden = getToolkitStorageKey('hide-memos', true);
+  const label = notHidden ? l10n('app.show', 'is-checked') : l10n('app.hide', 'not-checked');
+
+  const toggleHidden = () => {
+    toggleHiddenState(!notHidden);
+    controllerLookup('application').send('closeModal');
+    document.getElementById('ember1434').remove();
```

Comment:
we shouldn't need to do this, we're injecting into their modal so once you call `.send('closeModal')` we should unmount safely...maybe react is throwing a warning because it's not the one unmounting?

Reply:
sorry, you saw the wrong PR, i reverted this and messaged you about it.  I actually check if the div exists and destroy it before the return of a new one is created.  I was getting multiple divs without this, as each time the model window would open it would create a new div.  if you don't check to destroy before returning the div, and rely soly on it being destroyed by closeModel, then it won't be destroyed if the user opens "View" and selects " Show Running Balance."  The window will close, with the div remaining in the model, and a new div will be created when "View"is clicked again.

Reply:
it's all moo anyways, apparently the destroy method is not needed and something else was duplicating my model last night.